Application/EdFi.Ods.Admin/Initialization/InitializationEngine.cs

```diff
@@ -79,23 +80,23 @@ public void CreateUsers()
             }
         }
 
-        private void CreateUser(UserInitializationModel user)
+        private async void CreateUser(UserInitializationModel user)
```

Comment:
We shouldn't be using async void, this should return a Task, and then resharper is likely to tell you to just return the Task directly from SetDefaultVendorOnUserFromEmailAndNameAsync instead of awaiting it.  I'm guessing you were trying to avoid async from leaking out, but this is another good reason to handle the conversion to use async separately.

Reply:
done